pkl-core/src/main/java/org/pkl/core/EvaluatorImpl.java

```diff
@@ -67,6 +70,8 @@ public class EvaluatorImpl implements Evaluator {
   protected final BufferedLogger logger;
   protected final PackageResolver packageResolver;
```

Comment:
Now that we don't have a subclass, let's change all these fields to `private`, and add modifier `final` to class `EvaluatorImpl`